### PR TITLE
Refactor or to avro type function

### PR DESCRIPTION
**Pull request by buptyzc**
In some cases, the avro serialization process throw exception for type confused. I thind the root of the problem is that, we make a hard conversion from or type to avro type.That is right but not safe in prod environment. So we change the conversion and make it base on avro schema. Our prod env don't throw that exception again.
By the way , I check the code in open-replicator code about TimeColumn. we found the 1.0.7 version don't have the problem in dbus note.The open-replicator code like this:
"
    	public static java.sql.Time toTime(int value) {
		final int s = (int)(value % 100); value /= 100;
		final int m = (int)(value % 100);
		final int h = (int)(value / 100);
		final Calendar c = Calendar.getInstance();
        c.set(1970, 0, 1, h, m, s);
        c.set(Calendar.MILLISECOND, 0);
        return new java.sql.Time(c.getTimeInMillis());
	}
"

### CODE REVIEW

File: databus-util-cmdline/databus-util-cmdline-impl/src/main/java/com/linkedin/databus/util/FieldToAvro.java [hunk 1]
```diff
@@ -202,7 +202,7 @@ else if (typeInfo instanceof CollectionTypeInfo)
     field.put("default", null);
 
     // Field type
-    String[] type = new String[] { "null", typeInfo.getPrimitiveType().getAvroType() };
+    String[] type = new String[] { typeInfo.getPrimitiveType().getAvroType(), "null"};
```
Comment (chavdar) [inline databus-util-cmdline/databus-util-cmdline-impl/src/main/java/com/linkedin/databus/util/FieldToAvro.java]: This is a significant change. Putting the "null" type second means that "null" cannot be the default value anymore.

Why is this necessary.

Comment (buptyzc) [inline databus-util-cmdline/databus-util-cmdline-impl/src/main/java/com/linkedin/databus/util/FieldToAvro.java]: with using contains function , I think this change can make it run faster. I test the 'column is null case',and it runs right. I didn't consider the default value case ,I commit and change it back.Thank you.
